memgpt/utils.py

```diff
@@ -259,3 +259,11 @@ def get_schema_diff(schema_a, schema_b):
     difference = [line for line in difference if line.startswith("+ ") or line.startswith("- ")]
 
     return "".join(difference)
+
+
+def fix_file_path(path):

```

Comment:
@agiletechnologist instead of using `fix_file_path`, I think we should be using the tools in `os.path` which should automatically use the correct native system path format.

In this case, it looks like this is only applied on top of an `os.path.join` - does something break if this isn't done additionally?

Reply:
I must move that out of the utils and put it in the memgptlog.py. There is a bug related to config import on Windows when it is saved. 
There is a bug when loading the configuration for Windows paths with logging.